public/scripts.js

```diff
@@ -0,0 +1,81 @@
+const boxArray = [];
+let i = 1;
+
+const onChange = (e, i) => {
+  i = i - 1;
+
+  if (e.target.checked) {
+    const someStringToCheck = e.currentTarget.id;
+    const indexOfSpace = someStringToCheck.indexOf("-");
+    const index = parseInt(
+      someStringToCheck.substr(indexOfSpace + 1, someStringToCheck.length + 1)
+    );
+    document.getElementById(`box-${i}`);
+    let heightValue = document.getElementById(`height-${index}`).value + "px";
+    let WidthValue = document.getElementById(`width-${index}`).value + "px";
+    console.log("checkbox number -" + index);
```

Comment:
a co to za console log w produkcyjnym kodzie?! logi tylko podczas developmentu, jak coś publikujesz to logów nigdy nie dawaj. Chyba, że piszesz bibliotekę dla programistów.